pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "multi-imbalance"
+description = "Python package for tackling multiclass imbalance problems."
+version = "0.1.0"
+maintainers = [
+    {name = "Damian Horna, Kamil Pluciński, Hanna Klimczak, Jacek Grycza", email = "horna.damian@gmail.com"}
```

Comment:
I think we can add a few names here, of course, if you want ;)

Reply:
Done. I added 4 new names :)